modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -0,0 +1,256 @@
+    - grpc-v2
+  supportedModelFormats:
+    - autoSelect: true
+      name: openvino_ir
+      version: opset13
+    - name: onnx
+      version: "1"
+    - autoSelect: true
+      name: tensorflow
+      version: "1"
+    - autoSelect: true
+      name: tensorflow
+      version: "2"
+    - autoSelect: true
+      name: paddle
+      version: "2"
+    - autoSelect: true
+      name: pytorch
+      version: "2"
+----
+
+.. If you are using the OpenVINO Model Server example above, ensure that you insert the correct values required for any placeholders in the YAML code.
+.. Save the file with an appropriate file name.
+.. Apply the file containing your serving run time:
++
+----
+$ oc apply -f <serving run time file name> -n <namespace>
+----
+
+. Create an InferenceService custom resource (CR). Create a YAML file to contain the InferenceService CR. Using the OpenVINO Model Server example used previously, here is the corresponding YAML code:
+
++
+----
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true"
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    serving.kserve.io/deploymentMode: RawDeployment
+  name: <InferenceService-Name>
+spec:
+  predictor:
+    scaleMetric:
+    minReplicas: 1
+    scaleTarget:
+    canaryTrafficPercent:
+    serviceAccountName: <serviceAccountName>
+    model:
+      env: []
+      volumeMounts: []
+      modelFormat:
+        name: onnx
+      runtime: ovms-runtime
+      storageUri: s3://<bucket_name>/<model_directory_path>
+      resources:
+        requests:
+          memory: 5Gi
+    volumes: []
+----
+
+.. In your YAML code, ensure the following values are set correctly:
+
+* `serving.kserve.io/deploymentMode` must contain the value `RawDeployment`.
+* `modelFormat` must contain the value for your model format, such as `onnx`.
+* `storageUri` must contain the value for your model s3 storage directory, for example `s3://<bucket_name>/<model_directory_path>`.
+* `runtime` must contain the value for the name of your serving runtime, for `example,ovms-runtime`.
+
+.. Save the file with an appropriate file name.
+.. Apply the file containing your InferenceService CR:
++
+----
+$ oc apply -f <InferenceService CR file name> -n <namespace>
+----
+
+. Verify that all pods are running in your cluster:
+
++
+----
+$ oc get pods -n <namespace>
+----
+Example output:
+
++
+----
+NAME READY STATUS RESTARTS AGE 
+<isvc_name>-predictor-xxxxx-2mr5l 1/1 Running 2 165m
+console-698d866b78-m87pm 1/1 Running 2 165m
+----
+
+. After you verify that all pods are running, forward the service port to your local machine:
+
++
+----
+$ oc -n <namespace> port-forward pod/<pod-name> <local_port>:<remote_port>
+----
+Ensure that you replace `<namespace>`, `<pod-name>`, `<local_port>`, `<remote_port>` (this is the model server port, for example, `8888`) with values appropriate to your deployment.
```

Comment:
Need a "+" after the "----" ?

Reply:
As discussed, this change is not necessary.